addons/sale_management/report/sale_report_templates.xml

```diff
@@ -32,7 +32,7 @@
                                     t-att-style="option.discount and 'text-decoration: line-through' or None"
                                     t-att-class="option.discount and 'text-danger' or None"/>
                                 <div t-if="option.discount">
-                                    <t t-out="'%.2f' % ((1-option.discount / 100.0) * option.price_unit)"/>
+                                    <t t-out="((1-option.discount / 100.0) * option.price_unit)" t-options='{"widget": "monetary", "display_currency": doc.pricelist_id.currency_id}'/>
```

Comment:
```suggestion
                                    <t t-out="((1-option.discount / 100.0) * option.price_unit)" t-options='{"widget": "monetary", "display_currency": doc.currency_id}'/>
```

Won't change anything in this version, but has to be done from 16.2+ (#84920) so we might as well do it directly here.